adapters/richaudience/richaudience.go

```diff
@@ -160,18 +168,18 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 	bidResponse.Currency = bidResp.Cur
 
 	for _, reqBid := range bidReq.Imp {
-		for _, seatBid := range bidResp.SeatBid {
-			for i := range seatBid.Bid {
+		for j := range bidResp.SeatBid {
+			for i := range bidResp.SeatBid[j].Bid {
 
-				bidType := getMediaType(seatBid.Bid[i].ImpID, reqBid)
+				bidType := getMediaType(bidResp.SeatBid[j].Bid[i].ImpID, reqBid)
 
 				if bidType == "video" {
-					seatBid.Bid[i].W = reqBid.Video.W
-					seatBid.Bid[i].H = reqBid.Video.H
+					bidResp.SeatBid[j].Bid[i].W = reqBid.Video.W
+					bidResp.SeatBid[j].Bid[i].H = reqBid.Video.H
 				}
 
 				b := &adapters.TypedBid{
-					Bid:     &seatBid.Bid[i],
+					Bid:     &bidResp.SeatBid[j].Bid[i],
 					BidType: bidType,
 				}
```

Comment:
Why are these changes necessary for the bid processing loop? I'm not seeing the issue with the original code.

Reply:
There are no issues with this. We discussed this with the team and decided this is the right way to do this, similar to how we access Bid by index. I can revert it back.